gnocchi/rest/aggregates/api.py

```diff
@@ -14,7 +14,7 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-import fnmatch
+from distutils import util
```

Comment:
Please dont import distutils here but use gnocchi.rest.api.get_bool_param as already used in this file.

Reply:
sure, thanks.